pkg/gitreceive/slug_builder_info.go

```diff
@@ -19,12 +21,12 @@ type SlugBuilderInfo struct {
 
 // NewSlugBuilderInfo creates and populates a new SlugBuilderInfo based on the given data
 func NewSlugBuilderInfo(appName string, shortSha string, disableCaching bool) *SlugBuilderInfo {
-	slugName := fmt.Sprintf("%s:git-%s", appName, shortSha)
-	tarKey := fmt.Sprintf("home/%s/tar", slugName)
+	basePath := fmt.Sprintf(GitKeyPattern, appName, shortSha)
+	tarKey := fmt.Sprintf("%s/tar", basePath)
```

Comment:
I just changed around patter so the tar key could be generated by the cleaner. That meant the slugName was no longer was the correct meaning... since it contained the entire path, so I renamed it.